internal/clients/team4/client.go

```diff
@@ -87,8 +87,8 @@ type internalParameters struct {
 	agentsTrust   []float64
```

Comment:
I'll have some merge conflicts, but shouldn't be too bad to deal with them. I'll think about creating a type for agent trust and storing it in a map and having a function to get the trust array or something

Reply:
Sure thing